src/main/java/org/team1540/robot2025/subsystems/intake/IntakeIOReal.java

```diff
@@ -0,0 +1,122 @@
+package org.team1540.robot2025.subsystems.intake;
+
+import com.ctre.phoenix6.StatusCode;
+import com.ctre.phoenix6.configs.MotionMagicConfigs;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.revrobotics.spark.SparkBase;
+import com.revrobotics.spark.SparkLowLevel;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.SparkBaseConfig;
+import com.revrobotics.spark.config.SparkMaxConfig;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class IntakeIOReal implements IntakeIO {
+
+    // rotation of horizontal beams for the intake, clockwise to intake, counter-clockwise to spit out
+    private final TalonFX spinFalcon = new TalonFX(IntakeConstants.SPIN_FALCON_ID);
+
+    // controls position of intake
+    private final TalonFX pivotFalcon = new TalonFX(IntakeConstants.PIVOT_FALCON_ID);
+
+    private final MotionMagicVoltage pivotPositionRequest = new MotionMagicVoltage(0).withSlot(0);
+
+    // clockwise to intake, counter-clockwise to spit out
+    private final SparkMax funnelNeo = new SparkMax(IntakeConstants.NEO_ID, SparkLowLevel.MotorType.kBrushless);
+
+    private StatusCode spinFalconStatus;
+
+    private StatusCode pivotFalconStatus;
+
+    public IntakeIOReal() {
+
+        TalonFXConfiguration spinTalonFXConfigs = new TalonFXConfiguration();
+        TalonFXConfiguration pivotTalonFXConfigs = new TalonFXConfiguration();
+        SparkMaxConfig funnelNEOConfig = new SparkMaxConfig();
+
+        spinTalonFXConfigs.CurrentLimits.withStatorCurrentLimitEnable(true);
+        spinTalonFXConfigs.CurrentLimits.withStatorCurrentLimit(120);
+        spinTalonFXConfigs.CurrentLimits.withSupplyCurrentLimit(70);
+        spinTalonFXConfigs.CurrentLimits.withSupplyCurrentLowerLimit(40);
+        spinTalonFXConfigs.CurrentLimits.withSupplyCurrentLowerTime(0.5);
+        spinTalonFXConfigs.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+        spinTalonFXConfigs.MotorOutput.NeutralMode = NeutralModeValue.Coast;
+
+        Slot0Configs spinConfigs = spinTalonFXConfigs.Slot0;
+
+        spinFalcon.getConfigurator().apply(spinConfigs);
```

Comment:
This will only apply the motor PID configs, and none of the others. What are you trying to do here? You need to set `spinTalonFXConfigs.Slot0`, not create a new Slot0Configs. Also, you do not need PID gains on the spin motor, so this is completely unnecessary.